File: src/main/java/kr/mjc/rachel/basics/jdbc/movie/ListMovies.java
package kr.mjc.rachel.basics.jdbc.movie;

import lombok.extern.slf4j.Slf4j;

import java.util.List;
import java.util.Scanner;

@Slf4j
public class ListMovies {
  public static void main(String[] args) {
    System.out.print("count page : ");
    Scanner scanner = new Scanner(System.in);
    int count = scanner.nextInt();
    int page = scanner.nextInt();
    scanner.close();

    MovieDao movieDao = new MovieDaoUsingJdbcHelper();
    List<Movie> movieList = movieDao.listMovies(count, page);
    log.debug(movieList.toString());
  }
}

File: src/main/kotlin/kr/mjc/jacob/midterm/movie/ListMovies.kt
package kr.mjc.jacob.midterm.movie

import kr.mjc.jacob.jdbc.Page
import kr.mjc.jacob.midterm.applicationContext
import java.util.*

fun main() {
  val movieDao = applicationContext.getBean(MovieDao::class.java)

  print("pageNumber pageSize ? ")
  val page = Scanner(System.`in`).use {
    Page(it.nextInt(), it.nextInt())
  }

  val list = movieDao.findAll(page)
  list.forEach(::println)
}